tests/input_files/consinter3D_tsi_hex27_hex27.dat

```diff
@@ -105,7 +105,7 @@ TEKO_XML_FILE                   xml/block_preconditioner/thermo_solid.xml
 AZREUSE                         10
 -------------------------------------------------------------------MATERIALS
 MAT 1 MAT_Struct_ThrStVenantK YOUNGNUM 1 YOUNG 2.1e08 NUE 0.0 DENS 7.85e-06 THEXPANS 1.1e-05 CAPA 0.0 CONDUCT 1030.0 INITTEMP 273.15
-MAT 2 THERM_FourierIso CAPA 0.0 CONDUCT 1030.0
+MAT 2 MAT_Fourier CONDUCT_PARA_NUM 1 CAPA 0.0 CONDUCT 1030.0
```

Comment:
Is it possible to induce `CONDUCT_PARA_NUM` from the number of parameters since it's an inherent property? If it's not possible due to the logic of the code, then leave it as it is.